### ISSUE TITLE
Fullscreen 

### ISSUE THREAD

**Issue by sliceanddice9** [kiwibrowser/src.next#879]
Kiwi needs an optional built-in default fullscreen mode that does just that - fills the entire screen. This includes the removal of the toolbar. No third party, fullscreen extensions from the Chrome Store or GitHub are good enough.

**Comment by allytiago** [comment]
I already requested this function previously (#160) and what they gave me was a code to activate it through DevTools.  I particularly found a non-viable solution due to having more than one process to activate, in addition to having to keep the code always annotated.

 I got the code `javascript:document.documentElement.webkitRequestFullScreen();` which in particular does not solve the problem, since the function of a shortcut is to facilitate, something that does not exist here.

 My solution was to look for extensions.  I found this one to work, but it doesn't bring the well-known F11 shortcut that I so wanted Kiwi to bring.  And well, as I said, I received a javascript code to put in DevTools, so I can believe that it won't be difficult for the team to create a proprietary extension with the line of code and with support for the F11 shortcut, since the extension would change the code and insert the code mentioned above, avoiding the process of opening devtools, going to the console, inserting the code and hitting enter.

 The alternative extension I found was "[Full Screen Button](https://chrome.google.com/webstore/detail/full-screen-button/iokdoigdonjidoklkaonhielnkamibnp)"

**Comment by allytiago** [comment]
This function was previously requested (#160) and what they passed was a code to activate it through DevTools.  I particularly found a non-viable solution due to having more than one process to activate, in addition to having to keep the code always annotated.

 The code was: `javascript:document.documentElement.webkitRequestFullScreen();`  which in particular does not solve the problem, since the function of a shortcut is to facilitate, something that does not exist here.

 My solution was to look for extensions.  I found this one to work, but it doesn't bring 3the well-known F11 shortcut that I so wanted Kiwi to bring.  And well, as I said, I received a javascript code to put in DevTools, so I can believe that it won't be difficult for the team to create a proprietary extension with the line of code and with support for the F11 shortcut, since the extension would change the code and insert the code mentioned above, avoiding the process of opening devtools, going to the console, inserting the code and hitting enter.

 The alternative extension I found was "[Full Screen Button](https://chrome.google.com/webstore/detail/full-screen-button/iokdoigdonjidoklkaonhielnkamibnp)"

**Comment by sliceanddice9** [comment]
> javascript:document.documentElement.webkitRequestFullScreen();

If I want to try input the code myself how and where do I enter `javascript:document.documentElement.webkitRequestFullScreen();` in DevTools on Kiwi? The UI for the DevTools on Kiwi is very confusing. 

Yeah I have tried that "Full Screen Button" extension from the Chrome Store but the problem is enabling the full screen button itself on websites like [archive.org](http://archive.org/) and [rumble.com](http://rumble.com/), doesn't actually keep the website in fullscreen when navigating through the website by clicking links within the website to get from one place to the next.

Like for example: just say I'm on the home page of [rumble.com](http://rumble.com/)  after clicking the full screen button to activate the fullscreen action - I then proceed to tap on 'My Feed' or 'Watch History' or 'Trending' or any other button link on the Rumble website; as soon as that button link is pressed it immediately exits out from fullscreen mode and back into regular web browser mode with the toolbar up top, so in order to get back to fullscreen mode I need to tap the full screen button again. This happens every time I navigate the website by clicking any button links within the website itself which is quite annoying and tedious.
